src/SonarScanner.MSBuild.Common/SonarProperties.cs

```diff
@@ -52,6 +52,7 @@ public static class SonarProperties
         public const string PullRequestBase = "sonar.pullrequest.base";
         public const string PullRequestCacheBasePath = "sonar.pullrequest.cache.basepath";
         public const string WorkingDirectory = "sonar.working.directory";
+        public const string WorkingTemporaryDirectory = "sonar.working.temp.directory";
```

Comment:
We'll need to discuss the name for this.

Reply:
Agreed, I don't think the name as it currently is, represents this well. Perhaps something like `sonar.working.cache.directory` is better suited. I will go with the guidance from the code owners here.

Reply:
I think 'working' may be redundant here. How about `sonar.cache.directory` or `sonar.plugin.cache.directory`?